### PR TITLE
Handle more events

### PR DESCRIPTION
**Pull request by aduffeck**
We fixed an error in the audit service when adding space members and added logging for more events.

Fixes https://github.com/owncloud/ocis/issues/5029

### CODE REVIEW

File: services/audit/pkg/types/types.go [hunk 1]
```diff
@@ -205,6 +205,31 @@ type AuditEventSpaceDeleted struct {
...
+	GranteeUserID  string
+	GranteeGroupID string
+}
+
+// AuditEventSpaceUpdated is the event logged when a space is updated
+type AuditEventSpaceUpdated struct {
+	AuditEventSpaces
+
+	Name          string
+	Opqaue        map[string]string
```
Comment (micbar) [inline services/audit/pkg/types/types.go:229]: typo?

File: services/audit/pkg/types/types.go [hunk 2]
```diff
@@ -205,6 +205,31 @@ type AuditEventSpaceDeleted struct {
...
+
+	GranteeUserID  string
+	GranteeGroupID string
+}
+
+// AuditEventSpaceUpdated is the event logged when a space is updated
+type AuditEventSpaceUpdated struct {
+	AuditEventSpaces
+
+	Name          string
```
Comment (micbar) [inline services/audit/pkg/types/types.go:228]: Can we also add the updated description?

Comment (micbar) [reply services/audit/pkg/types/types.go:228]: Checked it. It is inside the opaque map

Comment (aduffeck) [reply services/audit/pkg/types/types.go:228]: :+1:

**Review by butonic — APPROVED** [review]

**Pull request merged**